presto-main/src/main/java/io/prestosql/security/AccessControlManager.java

```diff
@@ -20,6 +20,7 @@
 import io.airlift.log.Logger;
```

Comment:
I would change commit message to:
```
Allow SystemAccessControl to provide to event listeners
```

> enhancing event driven communicaiton about
executing queries in the system to SystemLevelAccessControl

I would drop this.